src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -7,7 +7,7 @@
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The student index provided is invalid";
```

Comment:
MESSAGE_INVALID_STUDENT_DISPLAYED_INDEX

But if we remove messages in future, shouldn't be an issue